staging/src/k8s.io/kubectl/pkg/cmd/drain/drain.go

```diff
@@ -327,6 +325,14 @@ func (o *DrainCmdOptions) deleteOrEvictPodsSimple(nodeInfo *resource.Info) error
 	if warnings := list.Warnings(); warnings != "" {
 		fmt.Fprintf(o.ErrOut, "WARNING: %s\n", warnings)
 	}
+	if o.drainer.DryRun {
+		if len(list.Pods()) > 0 {
```

Comment:
No need for this check, for-range will handle that for you

Reply:
oh right, ok. dropped